androidlibrary_lib/src/androidTest/java/org/opendatakit/database/data/TableMetaDataEntriesTest.java

```diff
@@ -0,0 +1,141 @@
+package org.opendatakit.database.data;
+
+import android.os.Parcel;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+@SuppressWarnings("ALL")
+@RunWith(AndroidJUnit4.class)
+public class TableMetaDataEntriesTest {
+
+    public TableMetaDataEntries testMetaDataEntries;
+
+    private final KeyValueStoreEntry ENTRY_1 = new KeyValueStoreEntry();
```

Comment:
The general convention is that only constants are capitalized (things you expect to not need to change).

Please fix everywhere